### ISSUE TITLE
try Toni's dbuild beta (with disk-space improvement)

### ISSUE THREAD

**Issue by SethTisue** [scala/community-builds#970]
see https://github.com/scala/community-builds/issues/905#issuecomment-490954630

**Comment by SethTisue** [comment]
@cunei during extraction I'm seeing:

```
[scala-js-stubs] --== Extracting dependencies for scala-js-stubs ==--
[scala-js-stubs] Fetching info for reference "967c0c4e4331b3873257b8229e4a8dd805c4b851" from https://github.com/scala-js/scala-js-stubs.git
[scala-js-stubs] into /Users/tisue/community.211/clones-0.9.17-toni22/dd3e2ea1721596e5ad60f746eeda6b59f4868da0-scala-js-stubs
fatal: ambiguous argument '967c0c4e4331b3873257b8229e4a8dd805c4b851^{commit}': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
[scala-js-stubs] Reference "967c0c4e4331b3873257b8229e4a8dd805c4b851" resolves to 967c0c4e4331b3873257b8229e4a8dd805c4b851
```

and for other repos as well. perhaps the error is harmless? it doesn't seem to prevent the build from proceeding. but 0.9.16 didn't do this

**Comment by SethTisue** [comment]
closing for now since https://github.com/lightbend/dbuild/pull/221 remains open
